Maker-Agents-Suit-Module/OSMAgent/src/agent.ts

```diff
@@ -9,17 +9,11 @@ import axios from "axios";
 const API_ENDPOINT: string = "https://changelog.makerdao.com/releases/mainnet/1.9.10/contracts.json";
```

Comment:
this is a specific api endpoint which was given during the first conversation with MakerDAO team. but I see [this active endpoint](https://chainlog.makerdao.com/api/mainnet/active.json) in their [docs](https://docs.makerdao.com/deployment-addresses/multi-collateral-dai-public-releases)
all versions can be seen [here](https://chainlog.makerdao.com/api/index.json) so this 1.9.10 is a bit old. 
I don't know if you should change this to the active one, or maybe we can ask this to them.